src/Main.java

```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {
+        String result[] = new String[110];
+        int index = 0;
+        for (int i = 0; i <= 9; i++) {
+            for (int j = 0; j <= 9; j++) {
+                if (i + j < 10) {
+                    result[index++] = String.format("%d + %d = %d", i, j, i + j);
+                }
+                if (i - j > -1) {

```

Comment:
zero or more sounds better than more than minus one, thus I think >= 0 is better here.

Reply:
Done, change` if (i - j > -1)` to `if (i - j >= 0)`